mycroft/skills/intent_service.py

```diff
@@ -332,12 +373,23 @@ def _converse(self, utterances, lang, message):
                               if time.time() - skill[
                                   1] <= self.converse_timeout * 60]
 
+
+        # first check for system levl skills
+        tmp_active_skills = copy(self.active_skills)
```

Comment:
again, casting to a tuple may make more sense than making a copy

Reply:
And again, this should really have a lock around it